petab/lint.py

```diff
@@ -9,6 +9,7 @@
 import libsbml
 import numpy as np
 import pandas as pd
+import sympy as sp
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'sympy'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4885513)